corehq/motech/openmrs/handler.py

```diff
@@ -35,7 +36,7 @@ def send_openmrs_visit(requests, info, form_config, person_uuid, visit_datetime)
         visit_datetime=visit_datetime,
         values_for_concept={obs.concept: [obs.value.get_value(info)]
                             for obs in form_config.openmrs_observations
-                            if obs.value.get_value(info)},
+                            if obs.value.get_value(info)},  # TODO: form_question_values
```

Comment:
What's this todo? Naively it seems like form_question_values is already passed in with `info`.

Reply:
Thanks for pointing it out. It's a stale comment. Removed.